### PR TITLE
Add link to JIRA issues.redhat.com

### PR DESCRIPTION
**Pull request by john-dupuy**
Screenshot from 2020-07-29 15-05-23

### CODE REVIEW

File: frontend/src/app.js [hunk 1]
```diff
@@ -343,6 +343,7 @@ class App extends React.Component {
                 <TextListItem component="dd"><a href="https://docs.ibutsu-project.org/" target="_blank" rel="noopener noreferrer">docs.ibutsu-project.org</a></TextListItem>
                 <TextListItem component="dt">Report bugs</TextListItem>
                 <TextListItem component="dd"><a href="https://github.com/ibutsu/ibutsu-server/issues/new" target="_blank" rel="noopener noreferrer">Submit an upstream issue</a></TextListItem>
+                <TextListItem component="dd"><a href="https://issues.redhat.com/secure/CreateIssue!default.jspa" target="_blank" rel="noopener noreferrer">Submit a JIRA issue</a></TextListItem>
```
Comment (rsnyman) [inline frontend/src/app.js:346]: Is this correct?

`CreateIssue!default.jspa`

Comment (john-dupuy) [reply frontend/src/app.js:346]: @rsnyman It works for me -- takes me to this page:

Screenshot from 2020-08-04 16-19-18

Comment (john-dupuy) [reply frontend/src/app.js:346]: Ah, but the project is set by default -- let me see if there's a way to specify that

Comment (john-dupuy) [reply frontend/src/app.js:346]: I guess it should be https://issues.redhat.com/secure/CreateIssue.jspa?pid=12324624&issuetype=1

**Pull request merged**